python/dgl/graphbolt/impl/csc_sampling_graph.py

```diff
@@ -760,13 +760,16 @@ def from_dglgraph(g: DGLGraph, is_homogeneous=False) -> CSCSamplingGraph:
     # Assign edge type according to the order of CSC matrix.
     type_per_edge = None if is_homogeneous else homo_g.edata[ETYPE][edge_ids]
 
+    # Assign edge attributes according to the original eids mapping.
+    edge_attributes = {ORIGINAL_EDGE_ID: homo_g.edata[EID][edge_ids]}
```

Comment:
Should we add option in the from_dglgraph and default it to false, such that if no eid needed in the task, we can avoid the memory cost for storing the eid,

Reply:
https://github.com/dmlc/dgl/issues/6406 issue filed